chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/rules/developer/DeveloperEditTransparencyAttestationValidation.java

```diff
@@ -46,12 +47,14 @@ private static boolean isTransparencyAttestationUpdated(final DeveloperDTO origi
             return true;
         } else {
             for (DeveloperACBMapDTO originalMapping : original.getTransparencyAttestationMappings()) {
-                for (DeveloperACBMapDTO changedMapping : changed.getTransparencyAttestationMappings()) {
-                    if (!Objects.equals(originalMapping.getAcbName(), changedMapping.getAcbName())
-                            || !Objects.equals(originalMapping.getTransparencyAttestation(),
-                                    changedMapping.getTransparencyAttestation())) {
-                        return true;
-                    }
+                Optional<DeveloperACBMapDTO> matchingMapping =
+                        changed.getTransparencyAttestationMappings().stream()
+                    .filter(changedMapping -> StringUtils.equals(originalMapping.getAcbName(), changedMapping.getAcbName()))
+                    .findFirst();
+                if (!matchingMapping.isPresent()
+                        || !StringUtils.equals(originalMapping.getTransparencyAttestation(),
```

Comment:
I believe there was a reason I used Objects.equals vs StringUtils.equals in the way null comparisons were done (specifically when both items were null). Do the relevant JUnits still pass with this code? Specifically here: https://github.com/chpladmin/chpl-api/pull/828/commits/6c63b0dcb6f432eed3b0b882f84e7197a08d4a87#diff-c352826099c6157d2b675926ef7bf2f7